### PR TITLE
[ru]: add Dutch National Flag Sort algo

### PR DESCRIPTION
**Pull request by romankurnovskii**

### CODE REVIEW

File: ru/Алгоритмы Сортировки/Флаг Нидерландов.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#### Постановка задачи
+
+Дан массив с несколькими цветами (красный, белый и синий), нужно упорядочить его таким образом, чтобы все цвета были сгруппированы вместе. Мы будем использовать целые числа `0`, `1` и `2` для представления красного, белого и синего цветов соответственно. 
+
+Нужно отсортировать массив на месте без использования дополнительной памяти и стандартной функции сортировки.
+
+#### Алгоритм
+
+1. Создайте три указателя - `low`, `mid` и `high`.
+1. Установите `low` и m`id в начало массива, а `high` в конец массива.
```
Comment (appgurueu) [inline ru/Алгоритмы Сортировки/Флаг Нидерландов.md:12]: it looks like you forgot to close a `monospace` here?

**Review by appgurueu — APPROVED** [review]
Looks okay content-wise (judging by what Google Translate is giving me). Dunno if using `1.` for each ordered list item is fine by our MD style guide. @Panquesito7?

File: ru/Алгоритмы Сортировки/Флаг Нидерландов.md (hunk 1 above)
Comment (romankurnovskii) [reply ru/Алгоритмы Сортировки/Флаг Нидерландов.md:12]: fixed, thank you

**Review by appgurueu — APPROVED** [review]

File: ru/Алгоритмы Сортировки/Флаг Нидерландов.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Задача флага Нидерландов
+
+#### Постановка задачи
+
+Дан массив с несколькими цветами (красный, белый и синий), нужно упорядочить его таким образом, чтобы все цвета были сгруппированы вместе. Мы будем использовать целые числа `0`, `1` и `2` для представления красного, белого и синего цветов соответственно.
```
Comment (Panquesito7) [inline ru/Алгоритмы Сортировки/Флаг Нидерландов.md:5]:
```suggestion
Дан массив с несколькими цветами (красный, белый и синий), нужно упорядочить его таким образом, чтобы все цвета были сгруппированы вместе. Мы будем использовать целые числа `0`, `1` и `2` для представления красного, белого и синего цветов соответственно.
```

File: ru/Алгоритмы Сортировки/Флаг Нидерландов.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Нужно отсортировать массив на месте без использования дополнительной памяти и стандартной функции сортировки.
+
+#### Алгоритм
+
+1. Создайте три указателя - `low`, `mid` и `high`.
+1. Установите `low` и `mid` в начало массива, а `high` в конец массива.
+1. Сравните `nums[mid]` со значением `1`. 
+   - Если `nums[mid]` равен 1, просто увеличьте mid. 
+   - Если `nums[mid]` равен 0, поменяйте `nums[mid]` и `nums[low]`, а затем увеличьте `mid` и `low` на 1. 
+   - Если `nums[mid]` равен 2, поменяйте `nums[mid]` и `nums[high]`, а затем уменьшите `high` на 1.
```
Comment (Panquesito7) [inline ru/Алгоритмы Сортировки/Флаг Нидерландов.md:16]:
```suggestion
1. Сравните `nums[mid]` со значением `1`.
   - Если `nums[mid]` равен 1, просто увеличьте mid.
   - Если `nums[mid]` равен 0, поменяйте `nums[mid]` и `nums[low]`, а затем увеличьте `mid` и `low` на 1.
   - Если `nums[mid]` равен 2, поменяйте `nums[mid]` и `nums[high]`, а затем уменьшите `high` на 1.
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Just a few trailing whitespace lines.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🚀

**Pull request merged**